src/Microsoft.TestPlatform.Common/Utilities/TypesToLoadUtilities.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright(c) Microsoft Corporation.All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.Common.Utilities
+{
+    using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+    using Microsoft.VisualStudio.TestPlatform.ObjectModel.Utilities;
+
+    using System;
+    using System.Collections.Generic;
+    using System.Reflection;
+    using System.Linq;
+
+    internal static class TypesToLoadUtilities
+    {
+        public const string TypesToLoadAttributeFullName = "Microsoft.VisualStudio.TestPlatform.ObjectModel.Utilities.TypesToLoadAttribute";
```

Comment:
Why is it public? Needs to be private.

Reply:
It's `internal` by class. I am planning to use this in E2E tests.